raiden/ui/web/src/app/services/raiden.service.ts

```diff
@@ -169,6 +170,16 @@ export class RaidenService {
         );
     }
 
+    public getPaymentHistory(tokenAddress: string, targetAddress: string): Observable<PaymentEvent[]> {
```

Comment:
Please, make `targetAddress` optional, to allow retrieving the whole payments history